examples/images_contours_and_fields/pcolormesh_levels.py

```diff
@@ -54,8 +115,6 @@
 # don't overlap
 fig.tight_layout()
 
-plt.show()
```

Comment:
leave the show(), so that one can still run this from the terminal and see something pop up?